content/general/concepts/what-is-graphql/what-is-graphql.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'What is GraphQL?' # This is the name that appears on the hub page for this language. Pay attention to capitalization and punctuation!
+Description: 'GraphQL is an open-source query language.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'APIs'
+  - 'Servers'
+CatalogContent:
+  - 'paths/computer-science'
+  - 'paths/back-end-engineer-career-path'
+  - 'paths/front-end-engineer-career-path'
+---
+
+## History of GraphQL
+
+GraphQL was developed by Facebook in 2012 to address internal issues with their mobile applications. As an alternative to typical REST architecture, GraphQL became open-source in 2015. Consequently, GraphQL makes data transfers more efficient, network usage for Facebook's mobile applications was drastically reduced. As of now, GraphQL has attracted a large number of developers and is becoming increasingly popular for client-server communication.
```

Comment:
> GraphQL was developed by Facebook in 2012...

Let's link out to the official GraphQL site, https://graphql.org/, on the first use of "GraphQL" in the entry.